ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_core.cc

```diff
@@ -93,52 +89,57 @@ TFCore::getModelPath ()
  * @note	the model will be loaded
  * @return 0 if OK. non-zero if error.
  *        -1 if the pb file is not loaded.
- *        -2 if the input properties is different with model.
+ *        -2 if the input properties are different with model.
  *        -3 if the Tensorflow session is not created.
  */
 int
 TFCore::loadModel ()
 {
+  g_message ("Start to load model [%s]", model_path);
+
 #if (DBG)
   gint64 start_time = g_get_real_time ();
 #endif
+
+  Status status;
   GraphDef graph_def;
-  Status load_graph_status =
-      ReadBinaryProto (Env::Default (), model_path, &graph_def);
-  if (!load_graph_status.ok()) {
-    GST_ERROR ("Failed to load compute graph at '%s'", model_path);
+
+  status = ReadBinaryProto (Env::Default (), model_path, &graph_def);
+  if (!status.ok()) {
+    GST_ERROR ("Failed to read graph.\n%s", status.ToString().c_str());
     return -1;
   }
 
-  /* get input tensor */
-  std::vector <const NodeDef*> placeholders;
-  for (const NodeDef& node : graph_def.node ()) {
-    if (node.op () == "Placeholder") {
-      placeholders.push_back (&node);
-    }
+  /* validate input tensor */
+  if (validateInputTensor (graph_def)) {
+    GST_ERROR ("Input Tensor Information is not valid");
+    return -2;
   }
 
-  if (placeholders.empty ()) {
-    GST_WARNING ("No inputs spotted.");
-  } else {
-    GST_INFO ("Found possible inputs: %ld", placeholders.size ());
-    if (inputTensorValidation (placeholders)) {
-      GST_ERROR ("Input Tensor Information is not valid");
-      return -2;
-    }
+  /* get session */
+  status = NewSession (SessionOptions (), &session);
+  if (!status.ok()) {
+    GST_ERROR ("Failed to init new session.\n%s", status.ToString().c_str());
+    return -3;
   }
 
-  /* get session */
-  Status new_session_status = NewSession (SessionOptions (), &session);
-  Status session_create_status = session->Create (graph_def);
-  if (!new_session_status.ok () || !session_create_status.ok ()) {
-    GST_ERROR ("Create Tensorflow Session was Failed");
+  status = session->Create (graph_def);
+  if (!status.ok()) {
+    GST_ERROR ("Failed to create session.\n%s", status.ToString().c_str());
     return -3;
   }
+
+  /* prepare output tensor */
+  for (int i = 0; i < outputTensorMeta.num_tensors; ++i) {
+    output_tensor_names.push_back (outputTensorMeta.info[i].name);
+  }
+
 #if (DBG)
   gint64 stop_time = g_get_real_time ();
   g_message ("Model is loaded: %" G_GINT64_FORMAT, (stop_time - start_time));
 #endif
+
+  g_message ("Finished to load model.");
```

Comment:
Make it GST_INFO

Reply:
logs about start/end to load model are removed